modules/opensimplex/doc_classes/OpenSimplexNoise.xml

```diff
@@ -31,8 +31,12 @@
 			</argument>
 			<argument index="1" name="height" type="int">
 			</argument>
+			<argument index="2" name="noise_offset_x" type="int" default="0">
+			</argument>
+			<argument index="3" name="noise_offset_y" type="int" default="0">
+			</argument>
 			<description>
-				Generate a noise image in [constant Image.FORMAT_L8] format with the requested [code]width[/code] and [code]height[/code], based on the current noise parameters.
+				Generate a noise image in [constant Image.FORMAT_L8] format with the requested [member width] and [member height], based on the current noise parameters. If [member noise_offset_x] and [member noise_offset_y] values are specified, these offset values are used as the coordinates of the top-left corner of the generated noise.
```

Comment:
Method arguments aren't specifiable as `[member]`, so you need to use `[code]`:

```suggestion
				Generate a noise image in [constant Image.FORMAT_L8] format with the requested [code]width[/code] and [code]height[/code], based on the current noise parameters. If [code]noise_offset_x[/code] and [code]noise_offset_y[/code] are specified, these offset values are used as the coordinates of the top-left corner of the generated noise.
```

Reply:
Fixed, and I'm looking into the pre-commit hooks now. Thank you for the feedback and sorry for the churn.